src/views/Simulation/components/Scene/Minimap.js

```diff
@@ -1,67 +1,87 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-import React, { forwardRef } from 'react';
+import React, { forwardRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import {
   CenterFocusStrong as CenterFocusStrongIcon,
   ZoomIn as ZoomInIcon,
   ZoomOut as ZoomOutIcon,
+  InfoOutlined as InfoOutlinedIcon,
 } from '@mui/icons-material';
 import { Button, Paper, Stack } from '@mui/material';
 import { MinimapContainer } from '../../utils/MinimapContainer';
+import { ChartLegendCard } from '../ChartLegendCard';
+import { BUTTON_BACKGROUND_COLOR } from './styleConstants';
 
 export const Minimap = forwardRef(function Minimap({ sceneContainerRef }, canvasMinimapRef) {
+  const [chartLegendIsOpen, setChartLegendIsOpen] = useState(false);
+
   return (
     <Stack
+      direction="column-reverse"
```

Comment:
Is there a technical reason for using "column-reverse"? It doesn't make the code easier to read and maintain :smile: